mrtt-ui/src/components/SiteInterventions.js

```diff
@@ -87,13 +122,42 @@ function SiteInterventionsForm() {
         toast.error(language.error.submit)
       })
   }
+
+  const handleBiophysicalInterventionsOnChange = (event, intervention) => {
+    const biophysicalInterventionTypesCheckedCopy = biophysicalInterventionTypesChecked
```

Comment:
This isnt a copy, its just a pointer to the original reference. YOu can use a spread to get a shallow copy (which is probably good enough for this context)

Reply:
ahh ok will use the spread operator (and update this in other files too)